src/components/cars/cars.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import Car from './car';
+import Pagination from '../dataTable/pagination';
+
+
+export default class CarsList extends Component {
+	state = {cars: []}
```

Comment:
no state mutating
use constructor
constructor(){
  this.state = {
    cars: []
  }
}